### PR TITLE
Updating appengine_samples

### PR DESCRIPTION
**Pull request by domesticmouse**
This PR is still WIP, it is pending on landing https://github.com/dart-lang/appengine/pull/102 and then publishing an updated `appengine` package. I'll update the sample `pubspec.yaml` files once that is done.

/cc @jonasfj

### CODE REVIEW

**Comment by jonasfj** [comment]
I didn't even know we had an `appengine_samples` repository :)

Looks good though :)

**Comment by domesticmouse** [comment]
This is now ready to land. 

PTAL @kevmoo

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+.dart_tool/
 .packages
+# Remove the following pattern if you wish to check in your lock file
+pubspec.lock
+
+# Conventional directory for build outputs
+build/
+
+# Directory created by dartdoc
+doc/api/
```
Comment (kevmoo) [inline .gitignore]: new line!

File: clientserver/README.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ To run the sample locally follow the instructions at
 [package:appengine](https://github.com/dart-lang/appengine/blob/master/README.md)
 and additionally run
 
-    $ pub build web
+    $ webdev build
```
Comment (kevmoo) [inline clientserver/README.md]: make this `pub run build_runner build` – webdev isn't always installed

**Review by kevmoo — APPROVED** [review]
Holy cow, these are old!

File: .gitignore (hunk 1 above)
Comment (domesticmouse) [reply .gitignore]: Done

File: clientserver/README.md (hunk 2 above)
Comment (domesticmouse) [reply clientserver/README.md]: And done

**Review by kevmoo — APPROVED** [review]

**Comment by domesticmouse** [comment]
Please either squash/merge or give me access to do the same. Thanks!

**Pull request merged**